storage/remote/otlptranslator/prometheus/normalize_label.go

```diff
@@ -15,7 +15,7 @@ import (
 // Labels that start with non-letter rune will be prefixed with "key_".
 //
 // Exception is made for double-underscores which are allowed.
-func NormalizeLabel(label string) string {
+func NormalizeLabel(label string) string { //nolint:revive // exported.
```

Comment:
The issue here is that the name stutters - `normalize.NormalizeLabel`.
But I think this code is copied from OpenTelemetry so we shouldn't tinker with it.